Gemfile

```diff
@@ -8,6 +8,7 @@ end
 source 'https://rubygems.org'
 source 'https://rails-assets.org'
 
+gem 'activerecord-import', '~> 0.20.2'
```

Comment:
Reading through activerecord-import's documentation on [requiring](https://github.com/zdennis/activerecord-import/wiki/Requiring), I think I'd rather we use `require: false` here and, in the `db/seeds.rb` file, `require 'activerecord-import'`, rather than loading and making the gem available throughout the app.

Reply:
In `db/seeds.rb`, looks like we'd need to do:

```rb
require 'activerecord-import/base'
require 'activerecord-import/active_record/adapters/postgresql_adapter'
```